src/App.vue

```diff
@@ -0,0 +1,85 @@
+<template>
+  <div id="app">
+    <div id="app">
+        <div id="page">
+            <navbar>
+                <h1>Tutorial Feedback</h1>
+            </navbar>
+            <div id="container">
+                <router-view></router-view>
+            </div>
+            <div id="footer">
+                <p>University of Toronto
+                </p>
+            </div>
+        </div>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+    name: "app",
+    data() {
+        return {
+            msg: "Welcome to Your Vue.js App"
+        };
+    }
+};
+</script>
+
+<style>
```

Comment:
I don't like having the style inline for the main app. Leave it as `app.css`. Only use inline styles for specific components.